docs/sources/architecture/ring.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Ring
+weight: 1000
+---
+
+# Ring
+
+This document builds upon the information in the [Loki Architecture](./) page
+
+It explains what exactly is the ring, what problem does it solve and how it actually works.
+
+We use ring in Distributor and Ingester components of Loki.
+
+Loki in microservice mode usually can have multiple ingesters and distributors. This multiple instances of same component(ingester or distributor) forms a ring (more presicisly Consisten Hash Ring).
```

Comment:
```suggestion
Loki in microservice mode usually can have multiple ingesters and distributors. This multiple instances of same component(ingester or distributor) forms a ring (more precisely Consistent Hash Ring).
```

Reply:
I think ingester forms a ring not distributors, distributors knows about the ring (to send data correctly) and which ingester hold which tokens from the hash ring.

That's my view of it though.

Reply:
I think distributor also has its own ring. At least that's what I see. 

https://github.com/grafana/loki/blob/master/pkg/distributor/distributor.go#L70-L76